ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -371,6 +373,78 @@ internal sealed class ArchiWebHandler {
 			return response != null; // Steam API doesn't respond with any error code, assume any response is a success
 		}
 
+		public class Inventory {
+			public string id { get; set; }
+			public string classid { get; set; }
+			public string instanceid { get; set; }
+			public string amount { get; set; }
+			public int pos { get; set; }
+		}
+
+		internal async Task<List<Inventory>> GetInventory() {
+			List<Inventory> result = new List<Inventory>();
+			try {
+				string json = await WebBrowser.UrlGetToContent("http://steamcommunity.com/my/inventory/json/753/6", Cookie).ConfigureAwait(false);
+				JObject jobj = JObject.Parse(json);
+				IList<JToken> results = jobj.SelectTokens("$.rgInventory.*").ToList();
+				foreach (JToken res in results) {
+					result.Add(JsonConvert.DeserializeObject<Inventory>(res.ToString()));
+				}
+			} catch (Exception) {
+				//just return empty list on error
+			}
+			return result;
+		}
+
+		internal async Task<bool> SendTradeOffer(List<Inventory> itemsSend, string masterid,string token=null) {
+			string sessionID;
+			if (!Cookie.TryGetValue("sessionid", out sessionID)) {
+				return false;
+			}
+
+			JObject tradeoffer =
+				new JObject(
+					new JProperty("newversion",true),
```

Comment:
I'd suggest creating new class similar to current ones (such as ```SteamTradeOffer.cs```) and using Serialize/Deserialize instead of specifying fields like that.